packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -141,6 +143,17 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         this.setState({ collapsedNodes: newList });
     }
 
+    private selectionChanged(payload: { [key: string]: number }) {
+        const offset = this.props.viewRange.getOffset();
+        const rawTimeStamp = payload['timestamp'];
```

Comment:
Here you need to convert it the string to a number. If conversion fails in the code then ignore signal.